toolkit/scripts/pkggen.mk

```diff
@@ -26,7 +26,8 @@ pkggen_rpms     = $(call shell_real_build_only, find $(RPMS_DIR)/*  2>/dev/null
 cache_working_dir      = $(PKGBUILD_DIR)/tdnf_cache_worker
 parse_working_dir      = $(BUILD_DIR)/spec_parsing
 rpmbuilding_logs_dir   = $(LOGS_DIR)/pkggen/rpmbuilding
-rpm_cache_files        = $(call shell_real_build_only, find $(CACHED_RPMS_DIR))
+rpm_cache_cache_dir    = $(CACHED_RPMS_DIR)/cache
```

Comment:
@pawelwms Noticed you were modifying this bit, any issues with this change? (if so, its easy to move the precacher working dir somehwere else. The `find` command was picking up all the random files inside the worker chroot and causing headaches since it was looking for everything inside the root `CACHED_RPMS_DIR/` rather than `CACHED_RPMS_DIR/cache`.

Reply:
Ignore my changes. I've just reverted them and they were purely cosmetic anyway. I can easily restore them after this PR is merged. Thanks for asking!